### PR TITLE
Resizable UART, Auto-load Prebuilt ROM, UART Send Byte

### PR DESCRIPTION
**Pull request by zoul0813**
* Makes the UART Terminal resizable (rows, cols)
* Fixes EOL for UART Terminal (`\n` is treated as a new line)
* Add "Send Byte" with a Decimal value (useful for testing)
* Introduce `?r=latest` parameter
  - this is being introduced lazily for wider testing
  - once verified, we can dynamically update the URL to reflect the selected ROM
* handful of CSS adjustments

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -305,18 +305,32 @@ <h4>Ready?</h4>
...
+          </label>
+          <button id="uart-file-send">Send</button>
+          <input type="number" id="uart-char" value="21" style="width: 5ch" />
+          <input type="button" id="uart-char-send" value="Send Byte" />
+
+          <label>
+            Rows: <input type="number" value="80" style="width: 5ch;" id="uart-cols" />
+          </label>
+          <label>
+            Cols: <input type="number" value="24" style="width: 5ch;" id="uart-rows" />
```
Comment (Zeal8bit) [inline index.html:331]: The label should enclose the input in HTML
```
<label>Cols:</label>
<input type="number" value="24" style="width: 5ch;" id="uart-rows" />
```

Also, `uart-rows` and `uart-cols` are swapped :smile:

**Comment by Zeal8bit** [comment]
Thanks a lot for this PR! Looks good to know, I left a small comment about swapped columns and rows

File: index.html (hunk 1 above)
Comment (higginsd-kong) [reply index.html]: I swapped the Rows/Cols labels around - oops ;)

**Comment by Zeal8bit** [comment]
Thanks, it's merged!

**Pull request merged**